packages/server/modules/emails/services/verification.js

```diff
@@ -1,42 +1,41 @@
-const appRoot = require( 'app-root-path' )
-require( 'dotenv' ).config( { 'path': `${appRoot}/.env` } )
-const crs = require( 'crypto-random-string' )
+require('../../../bootstrap')
```

Comment:
Moved common entrypoint init code to `bootstrap.js` for better DRYness

Reply:
why r we requiring the env file here?

Reply:
i think this is a leftover thing, u can just delete the bootstrap require all together